test/compute/agent-node-config.test.ts

```diff
@@ -20,6 +20,7 @@ test('Agents Resource is present', () => {
   });
   const stack = new CIStack(app, 'TestStack', {
     env: { account: 'test-account', region: 'us-east-1' },
+    alternativeNodeConfig: [],
```

Comment:
Can you add one more test with actual config just to be sure it works as expected?

Reply:
Updated to confirm the existing agents exist and new behavior for alternative agent nodes.